k8s-install/config.yaml

```diff
@@ -0,0 +1,39 @@
+# This ConfigMap can be used to configure a self-hosted Canal installation.
+# See `canal.yaml` for an example of a Canal deployment which uses
+# the config in this ConfigMap.
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: canal-config 
+  namespace: kube-system
+data:
+  # Configure this with the location of your etcd cluster.
+  etcd_endpoints: "http://127.0.0.1:2379"
+
+  # The interface used by canal for host <-> host communication.
+  canal_iface: "eth1"
```

Comment:
This puts a big requirement on needing all hosts in the cluster to use "eth1" for host<-> communications. Could we at least make this optional?